### PR TITLE
Fix/various bugs

### LINKED ISSUE #11: Fix: bugs after updating dmss and dm-core

### PR DESCRIPTION
**Pull request by KristianKjerstad**
## What does this pull request change?
fi bugs.
Very big PR, but almost all changes are related to moving recipes out of blueprints
## Why is this pull request needed?
bugs
## Issues related to this change
closes #11

### CODE REVIEW

File: app/data/app_mooring_db/simpos/mooringLS/env/Environment.json [hunk 1]
```diff
@@ -24,54 +24,54 @@
...
-      "type": "system/SIMOS/BlueprintAttribute",
+      "type": "CORE:BlueprintAttribute",
       "description": "Current return period in years.",
       "default": "{\"name\": \"currentReturnPeriod\", \"description\": \"Current return period in years.\", \"type\": \"app_mooring_db/marmo/containers/DimensionalScalar\", \"label\": \"Current Return Period [year]\", \"unit\": \"-\", \"value\":10}"
     },
     {
       "name": "currentDirectionRelativeToWind",
       "attributeType": "app_mooring_db/marmo/containers/DimensionalScalar",
-      "type": "system/SIMOS/BlueprintAttribute",
+      "type": "CORE:BlueprintAttribute",
```
Comment (KristianKjerstad) [inline app/data/app_mooring_db/simpos/mooringLS/env/Environment.json:74]: remove recipes

File: app/data/app_mooring_db/simpos/mooringLS/analysis/Config.json [hunk 2]
```diff
@@ -1,62 +1,62 @@
...
+      "type": "CORE:BlueprintAttribute",
       "description": "simulation length in seconds.",
       "default": "{\"name\": \"simulationLength\", \"description\": \"simulation length in seconds for limit state analysis.\", \"type\": \"app_mooring_db/marmo/containers/DimensionalScalar\", \"label\": \"Simulation Length [s]\", \"unit\": \"s\", \"value\":50}"
     }
   ],
   "storageRecipes": [],
   "uiRecipes": [
     {
-      "type": "system/SIMOS/UiRecipe",
+      "type": "CORE:UiRecipe",
```
Comment (KristianKjerstad) [inline app/data/app_mooring_db/simpos/mooringLS/analysis/Config.json:51]: move recipes out

File: app/data/app_mooring_db/simpos/mooringLS/MooringLS.json [hunk 3]
```diff
@@ -1,78 +1,78 @@
...
+      "type": "CORE:BlueprintAttribute",
       "description": "Simulation configuration.",
       "attributeType": "app_mooring_db/simpos/sima/config/SimaAppService"
     }
   ],
   "storageRecipes": [],
   "uiRecipes": [
     {
-      "type": "system/SIMOS/UiRecipe",
+      "type": "CORE:UiRecipe",
```
Comment (KristianKjerstad) [inline app/data/app_mooring_db/simpos/mooringLS/MooringLS.json:60]: move recipes out

File: app/data/app_mooring_db/simpos/mooringLS/positioning/Condition.json [hunk 4]
```diff
@@ -1,46 +1,46 @@
...
-      "type": "system/SIMOS/BlueprintAttribute",
+      "type": "CORE:BlueprintAttribute",
       "description": "broken mooring line numbers, a comma separated list",
       "default": "{\"name\": \"brokenLines\", \"description\": \"A comma separated list of broken mooring line numbers, None: no broken line.\", \"type\": \"app_mooring_db/marmo/containers/SimpleString\", \"label\": \"Broken Lines\", \"value\":\"None\"}"
     },
     {
       "name": "lineNumbers",
       "attributeType": "app_mooring_db/marmo/containers/SimpleString",
-      "type": "system/SIMOS/BlueprintAttribute",
+      "type": "CORE:BlueprintAttribute",
```
Comment (KristianKjerstad) [inline app/data/app_mooring_db/simpos/mooringLS/positioning/Condition.json:43]: move recipes out

File: app/data/app_mooring_db/simpos/models/containers/TextFile.json [hunk 5]
```diff
@@ -1,44 +1,44 @@
...
+      "type": "CORE:BlueprintAttribute",
       "description": "file label",
       "attributeType": "string"
     }
   ],
   "storageRecipes": [],
   "uiRecipes": [
     {
-      "type": "system/SIMOS/UiRecipe",
+      "type": "CORE:UiRecipe",
```
Comment (KristianKjerstad) [inline app/data/app_mooring_db/simpos/models/containers/TextFile.json:41]: move recipes

File: app/data/app_mooring_db/simpos/models/singleEnv/Current.json [hunk 6]
```diff
@@ -38,21 +38,21 @@
...
-      "type": "system/SIMOS/BlueprintAttribute",
+      "type": "CORE:BlueprintAttribute",
       "description": "Current direction -going to- in [deg].",
       "default": "{\"name\": \"currentDirection\", \"description\": \"Current direction -going to- in [deg].\", \"type\": \"app_mooring_db/marmo/containers/DimensionalScalar\", \"label\": \"Current Direction [deg]\", \"unit\": \"deg\", \"value\":330.0}"
     },
     {
       "name": "currentVelocities",
       "attributeType": "app_mooring_db/marmo/containers/SimpleString",
-      "type": "system/SIMOS/BlueprintAttribute",
+      "type": "CORE:BlueprintAttribute",
```
Comment (KristianKjerstad) [inline app/data/app_mooring_db/simpos/models/singleEnv/Current.json:55]: move recipes

File: app/data/app_mooring_db/simpos/models/singleEnv/Environment.json [hunk 7]
```diff
@@ -53,7 +53,7 @@
 
     {
       "name": "Raw",
-      "type": "system/SIMOS/UiRecipe",
+      "type": "CORE:UiRecipe",
```
Comment (KristianKjerstad) [inline app/data/app_mooring_db/simpos/models/singleEnv/Environment.json:56]: move recipes

**Review by soofstad — APPROVED** [review]

**Pull request merged**